script/tasks/diamondMakeImmutable.sh

```diff
@@ -42,17 +41,14 @@ diamondMakeImmutable() {
   echo "Last chance: Do you want to abort?"
   gum confirm && exit 1 || echo "OK, let's do it"
 
-
-
   #---------------------------------------------------------------------------------------------------------------------
   echo "PART 1 - TRANSFER OWNERSHIP OF PERIPHERY CONTRACTS"
-	attempts=1
+  attempts=1
 
-  while [ $attempts -lt 11 ]
-  do
+  while [ $attempts -lt 11 ]; do
     echo "[info] >>>>>>>>>>>>>>>>>>>>>>>>>>>> Trying to transfer ownership of periphery contracts now - attempt ${attempts}"
     # try to execute call
-    RAW_RETURN_DATA=$(NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT") forge script script/tasks/solidity/TransferOwnershipOfPeripheryContractsForImmutable.s.sol -f $NETWORK -vvvv --json --silent --broadcast --verify --skip-simulation --legacy --tc DeployScript)
+    RAW_RETURN_DATA=$(NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX PRIVATE_KEY=$(getAccount "$NETWORK" "$ENVIRONMENT") forge script script/tasks/solidity/TransferOwnershipOfPeripheryContractsForImmutable.s.sol -f $NETWORK -vvvv --json --silent --broadcast --verify --skip-simulation --legacy --tc DeployScript)
```

Comment:
Seems wrong...

PRIVATE_KEY will only contain the name of the stored key, not the private key itself

I think we should differentiate the calls where we use --account (>> use getAccount helper function and add --password flag) and the calls where we need the actual privateKey (>> use getPrivateKey helper function and no --password flag)